woocommerce/woocommerce-android
# Issue/5801 simple payments link

Closes #5801 - adds a "Share payment link" row to the simple payment flow to enable users to easily share the link to taking payment. Note that this requires WooCommerce 6.4 for testing since the `payment_url` field wasn't available until that version.

This means either using the WooCommerce beta tester plugin to install the latest 6.4 RC, or downloading the RC [from here](https://developer.woocommerce.com/2022/03/29/woocommerce-6-4-rc/) and uploading it manually.

To test:

* Before updating to the 6.4 RC, step through the simple payment flow and verify the "Share payment link" does _not_ appear
* Update to the 6.4 RC and verify that it does show, and tapping it enables sharing the link

https://user-images.githubusercontent.com/3903757/163035904-7c144470-d8ed-4252-9146-2dde02928275.mp4
